server/src/main/java/org/elasticsearch/cluster/metadata/MetadataCreateDataStreamService.java

```diff
@@ -51,13 +55,16 @@
         new LinkedHashSet<>(List.of(DateFieldMapper.CONTENT_TYPE, DateFieldMapper.DATE_NANOS_CONTENT_TYPE));
 
     private final ClusterService clusterService;
+    private final IndicesService indicesService;
```

Comment:
I think this is unused?

Reply:
A left over from an iteration that never made it into the pr.